### ISSUE TITLE
save datasets in .h5

### ISSUE THREAD

**Issue by arnaudon** [ImperialCollegeLondon/hcga#51]
the pickle is not good for large datasets, it makes memory error. Investigating that, and maybe using .h5 to save processed dataset.

**Comment by peach-lucien** [comment]
yes i have been encountering this issue too... DD is now 1.1gb in size
